api/v1alpha3/condition_consts.go

```diff
@@ -29,6 +29,10 @@ const (
 	// DeletingReason (Severity=Info) documents an condition not in Status=True because the underlying object it is currently being deleted.
 	DeletingReason = "Deleting"
 
+	// DeletingFailedReason (Severity=Warning) documents an condition not in Status=True because the underlying object
+	// encountered problems during deletion. This is a warning because the reconciler will retry deletion.
+	DeletingFailedReason = "DeletingFailed"
```

Comment:
```suggestion
	// DeletionFailedReason (Severity=Warning) documents an condition not in Status=True because the underlying object
	// encountered problems during deletion. This is a warning because the reconciler will retry deletion.
	DeletionFailedReason = "DeletionFailed"
```

Reply:
done!